src/main/app/initialize.test.js

```diff
@@ -157,13 +158,17 @@ jest.mock('main/UserActivityMonitor', () => ({
     startMonitoring: jest.fn(),
 }));
 jest.mock('main/windows/windowManager', () => ({
-    getMainWindow: jest.fn(),
     showMainWindow: jest.fn(),
-    sendToMattermostViews: jest.fn(),
     sendToRenderer: jest.fn(),
     getServerNameByWebContentsId: jest.fn(),
     getServerURLFromWebContentsId: jest.fn(),
 }));
+jest.mock('main/windows/settingsWindow', () => ({
+    show: jest.fn(),
+}));
+jest.mock('main/windows/mainWindow', () => ({
+    get: jest.fn(),
+}));
```

Comment:
I'm seeing this repeating itself.. can you add the mock in a test setup file of sort? we have one in the mobile app that works pretty well.

Reply:
I can, but it's really only used in a few places, and we're mocking different methods in different spots as well. Some files are adding additional methods, and others are mocking that `get` always just returns a blank object (useful for methods just checking if the MainWindow exists or not).

I think we'd just be moving the code bloat around, and I do like to be deliberate about what I mock where, especially when the mocked result has some consequence, so I'm going to leave this one as is for now. Might revisit later.